scripts/getUrl.js

```diff
@@ -0,0 +1,2 @@
+var process = require('process');
+console.log('http://ns.useraddress.net:5300/v1/dns/' + process.argv[2].split('.').reverse().join('/') + '/_acme-challenge');
```

Comment:
We can replace this file with inline perl :)

`perl -lpe '$_ = join "/", reverse split /\./'`